db/migrate/20190917095127_add_state_to_settings.rb

```diff
@@ -0,0 +1,5 @@
+class AddStateToSettings < ActiveRecord::Migration[5.2]
```

Comment:
@chhga this cannot inherit from `ActiveRecord::Migration[5.2]`, please use `SpreeExtension::Migration` otherwise it won't work for older Spree versions

Reply:
@damianlegawiec fixed